Makefile

```diff
@@ -1,193 +1,36 @@
-# Short doc about the commands (please keep in sync with cmd/README for now):
-# ------------------------ User commands -----------------------
-# gnokey        Key manipulation, also general interaction with gnoland
-# gnoland       Runs the blockchain node
-# gnotxport     Importing/exporting transactions from local blockchain node storage
-# website       Serves gno website, along with user-defined content
-# logos         Intended to be used as a browser
-#
-# ---------------------- Developer commands -------------------
-# gnoscan       Dumps imports from specified file’s AST
-# genproto      Helper for generating .proto implementations
-# gnofaucet     Serves GNOT faucet
-# gnodev        Handy tool for developing gno packages & realms
+.PHONY: help
+help:
+	@echo "Available make commands:"
+	@cat Makefile | grep '^[a-z][^:]*:' | cut -d: -f1 | sort | sed 's/^/  /'
 
-# Dist suite
-.PHONY: logos goscan gnoland gnokey gnofaucet logos reset gnoweb gnotxport
-all: build
-
-build: gnoland gnokey gnodev goscan logos gnoweb gnotxport gnofaucet
-
-install: install_gnodev install_gnokey
-
-reset:
-	rm -rf testdir
-	make
-
-tools:
-	go build -o build/logjack ./pkgs/autofile/cmd
-
-gnoland:
-	@echo "Building gnoland"
-	go build -o build/gnoland ./cmd/gnoland
-
-gnokey:
-	@echo "Building gnokey"
-	go build -o build/gnokey ./cmd/gnokey
-
-gnodev:
-	@echo "Building gnodev"
-	go build -o build/gnodev ./cmd/gnodev
+rundep=go run -modfile ../misc/devdeps/go.mod
 
+# shortcuts to frequently used commands from sub-components.
 install_gnokey:
-	@echo "Installing gnokey"
-	go install ./cmd/gnokey
-
-install_gnodev:
-	@echo "Installing gnodev"
-	go install ./cmd/gnodev
-
-gnofaucet:
-	@echo "Building gnofaucet"
-	go build -o build/gnofaucet ./cmd/gnofaucet
-
-# goscan scans go code to determine its AST
-goscan:
-	@echo "Building goscan"
-	go build -o build/goscan ./cmd/goscan
-
-gnoweb:
-	@echo "Building website"
-
-	go build -o build/website ./gnoland/website
-
-gnotxport:
-	@echo "Building gnotxport"
-	go build -o build/gnotxport ./cmd/gnotxport
-
-logos:
-	@echo "building logos"
-	go build -o build/logos ./misc/logos/cmd/logos.go
-
-clean:
-	rm -rf build
-
-examples.precompile: install_gnodev
-	go run ./cmd/gnodev precompile --verbose ./examples
-
-examples.build: install_gnodev examples.precompile
-	go run ./cmd/gnodev build --verbose ./examples
-
-########################################
-# Formatting, linting.
-
-rundep=go run -modfile ./misc/devdeps/go.mod
+	go install ./gno.land/cmd/gnokey
```

Comment:
Up to you, I like both.

Reply:
`$(MAKE) -C` is already used in other targets below, so for consistency I think this makes sense.

Reply:
To sum up, the output can either have noise or silent directory changes making it more useful for multi-command processes rather than single ones. However, I will implement your recommendation and release an update soon.